eiffel-syntax-and-usage/event-design-guidelines.md

```diff
@@ -34,3 +34,32 @@ The design of Eiffel events is governed by the following guidelines:
 * __Key names in camelCase:__ All key names shall use camelCase.
 * __Enumerations in CAPS_WITH_UNDERSCORE:__ All enumerated values shall use CAPS_WITH_UNDERSCORE.
 * __Link types are nouns:__ Link types shall have names that fit the pattern "\<target event> is the \<source event>'s \<link type>", implying that they are nouns that describe the relationship between the source and the target event.
+
+## Benefits and Implications
+
+By following the design guidelines the Eiffel protocol provides a language that
+gives producer and consumer a common way of understanding and describing the
+occurrences in the system by defining a contract between them.
+
+Following the guidlines stated, they adress the following issues:
```

Comment:
```suggestion
Following the guidelines stated, they address the following issues:
```

Reply:
Fixed in [6d5312c](https://github.com/eiffel-community/eiffel/pull/305/commits/6d5312c07d053524364f139d5805cf1c1c0c60d8)